packages/core/src/text/Text.css

```diff
@@ -2,6 +2,7 @@
 .uitkText {
   font-size: var(--uitk-text-fontSize);
   line-height: var(--uitk-text-lineHeight);
+  color: var(--text-color);
```

Comment:
```suggestion
  color: var(--uitkText-color, var(--text-color));
```
?

Reply:
Not sure this would be safe. This will potentially break the primary/secondary variants.

Reply:
To me, the intention of these variables is to allow consumers to build on Text. They could still scope uitkText-color to the primary or secondary variant classes, but for users who want to just override they shouldn't have to override multiple tokens